src/helpers/coordinates.ts

```diff
@@ -45,6 +44,14 @@ export function lettersToNumber(letters: string): number {
   return result;
 }
 
+function isCharALetter(char: string) {
+  return (char >= "A" && char <= "Z") || (char >= "a" && char <= "z");
```

Comment:
since you are checking lower case here, I don't think you need `toUpperCase` below